.github/workflows/newissues2lounge.yml

```diff
@@ -0,0 +1,16 @@
+on:
+  issues:
+    types: opened
+
+jobs:
+  assign_issue_to_project:
+    runs-on: ubuntu-latest
+    name: Assign new issue to project
+    steps:
+    - name: Create new project card with issue
+      id: list
+      uses: qmacro/action-add-issue-to-project-column@v1
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        project: 'Issue Lounge'
```

Comment:
```suggestion
        project: 'Issue_Lounge'
```

Reply:
?

Reply:
I went the other way round and renamed the project itself